content/registries/_index.md

```diff
@@ -13,3 +13,61 @@ weight: 23
 oc patch image.config.openshift.io/cluster -p \
 '{"spec":{"allowedRegistriesForImport":[{"domainName":"my.own.registry.example.com:8888","insecure":true}],"registrySources":{"insecureRegistries":["my.own.registry.example.com:8888"]}}}' --type='merge'
 ```
+
+# Configure custom/insecure registry to search path in OCP 4.x
+
+At this moment, there seems to be no way to remove/modify or append any additional entries to unqualified-search-registries line in /etc/containers/registries.conf file as of now (OCP <=4.3).
+
+A workaround to modify the file is using a machineconfig that do the job that should be done by `image.config.openshift.io/cluster` (full control of the /etc/containers/{policy.json,registries.conf} content).
```

Comment:
Meanwhile, the current workaround is to modify the `/etc/containers/{policy.json,registries.conf}` files using a `machineconfig` object instead.